com.microsoft.mrtk.input/Interactors/Grab/GrabInteractor.cs

```diff
@@ -12,12 +14,17 @@ namespace Microsoft.MixedReality.Toolkit.Input
     [UnityEngine.AddComponentMenu("MRTK/Input/Grab Interactor")]
     public class GrabInteractor : HandJointInteractor, IGrabInteractor
     {
+        [SerializeReference]
+        [InterfaceSelector]
+        [Tooltip("The pose source representing the worldspace pose of the hand pinching point.")]
+        private IPoseSource pinchPoseSource;
+
         /// <summary>
         /// Get near interaction point from hands aggregator.
         /// </summary>
-        protected override bool TryGetInteractionPoint(out HandJointPose jointPose)
+        protected override bool TryGetInteractionPoint(out Pose pose)
         {
-            return HandsAggregator.TryGetPinchingPoint(HandNode, out jointPose);
+            return pinchPoseSource.TryGetPose(out pose);
```

Comment:
We might need to null check these `IPoseSource`s before use

Reply:
Can do

Reply:
Done